tsconfig-strict.json

```diff
@@ -0,0 +1,27 @@
+// FUTURE TBD remove this config in favor of
+// strict type checking for all sources in
+// tsconfig.json
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "target": "es5",
+    "module": "commonjs",
+    "resolveJsonModule": true,
+    "strict": true,
+    /* Additional Checks */
+    "noUnusedLocals": true,                /* Report errors on unused locals. */
+    "noUnusedParameters": true,            /* Report errors on unused parameters. */
+    "noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
+    "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
```

Comment:
These additional checks have nothing to do with type checking. We have ESLint for this. Please remove them.

Reply:
Gone